### ISSUE TITLE
New feature "Allow audio"? How to allow?

### ISSUE THREAD

**Issue by Teufelskomet** [thecodingmachine/workadventure#3071]
If the "audioVolume" property was enabled in a layer, then a volume slider appeared in WA.
Now the message "Allow sound" appears. But neither the space bar nor a mouse click nor enter leads to an activation.
How to allow the audio / sound?

**Comment by LindezaGrey** [comment]
Hi again,

the "allow audio" button not only appears when you have to give the website rights to play audio, but also when there is an error with your audio-configuration in the map. taking a glimpse at your map it seems that you added the property "audioVolume" as "bool" whereas it has to be a "float", see here:

https://workadventu.re/map-building/special-zones.md

> If the "audioVolume" property is set, the audio player uses either the value of the property or the last volume set by the user - whichever is smaller. This property is a float from 0 to 1.0

**Comment by LindezaGrey** [comment]
Hi again,

the "allow audio" button not only appears when you have to give the website rights to play audio, but also when there is an error with your audio-configuration in the map. taking a glimpse at your map it seems that you added the property "audioVolume" as "bool" whereas it has to be a "float", see here:

https://workadventu.re/map-building/special-zones.md

> If the "audioVolume" property is set, the audio player uses either the value of the property or the last volume set by the user - whichever is smaller. This property is a float from 0 to 1.0

**Comment by Teufelskomet** [comment]
Hi ,
the tip with the float property don't work. 
Now I delete the"audioVolume" property and again only "allow audio".

No one of my audios play.

As described in the documentation the "playAudio" property is located in a layer. 

This setup with "audioVolume" as bool and "playAudio" in a layer is not new.  This setup is working for many months. But I have never seen before the "allow sounds" message.

**Comment by LindezaGrey** [comment]
Then check your browser logs (F12 and then the "network" tab - at least in google chrome). Another issue that can cause the **allow audio** button to appear is, when the mp3 could not be loaded. I saw that sometimes the map refers to "/maps/Sounds/xxx.mp3" and sometimes just "Sounds/xxx.mp3". If a sound could not be loaded you can see it in **red** in your network tab with a status code 404. that means you have to adapt the path to the sound file in your map

**Comment by Teufelskomet** [comment]
it took me some time to find the cause.  There is one or more problems on my side.A few days ago my site was updated to the new WA standard.  Something went wrong in the process. 
**First**, something goes mighty wrong when updating my gh-pages in Github. When I change the master branch, everything in the gh-pages branch is deleted except the change. So I lose the index.html, but also all subdirectories and therefore also the folder with the sounds.

But even if I create the gh-pages branch from scratch from the master, I get the message "allow sounds". Although the file exists and the url in the property is correct.

In any case the error is on my side and not WA. Therefore I close the issue.

Oh yes, until I can fix the bugs I switched to the old version, see here:
https://play.workadventu.re/@/heitec/test/nowhere

There the property "audioVolume" is defined as bool. Seems to work as well as float
